packages/webapp/src/components/FilterPage/index.tsx

```diff
@@ -6,6 +6,24 @@ import { Underlined } from '../Typography';
 import { useTranslation } from 'react-i18next';
 import Button from '../Form/Button';
 import FilterGroup from '../Filter/FilterGroup';
```

Comment:
The data flow for the non-finance filters is:

1. Filter Container (Filter/Task, Filter/CropVariety, Filter/Documents etc.)
2. PureFilterPage
3. FilterGroup

The container defines the filterRef and the relationship with the Redux store. The PureFilterPage component is a layout component with buttons that control the overall clearing and applying (saving) of filters. The FilterGroup component displays each individual filter. Each controls their own update of the filterRef, and, through the passed onChange, dirties the parent filter state to allowing saving. The onApply handler passed from the container to the PureFilterPage saves the updated filterRef to the Redux store.